src/screens/DomainScreen/index.js

```diff
@@ -125,10 +126,12 @@ const DomainScreen = () => {
       }
 
       await getDomainFeed(postOffset);
+    } else {
+      await getDomainFeed(postOffset);
+    }
 
-      if (withLoading) {
-        setLoading(false);
-      }
+    if (withLoading) {
+      setLoading(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `getDomainFeed(postOffset)` has been duplicated. It's now being called regardless of whether the domain name matches the last selected domain or not. This seems redundant and could potentially lead to unnecessary network requests. If this is not intentional, consider removing the duplicate call.

```diff
- } else {
-   await getDomainFeed(postOffset);
- }
```

229:
The `await` keyword has been removed from the filter operation on the `ifollow` array. This is correct because `filter()` is a synchronous operation and doesn't return a promise, so there's no need to `await` its result.


<!-- This is an auto-generated comment by OSS CodeRabbit -->